CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 0.21.0-rc.0 / 2020-06-05
+
+* [CHANGE] [amtool] Remove default assignment of environment variables. #2161
+* [CHANGE] Enforce 512KB event size limit for Pagerduty events. #2225
+* [ENHANCEMENT] [amtool] Add `cluster` command to show cluster and peer statuses. #2256
+* [ENHANCEMENT] Add redirection from `/` to the routes prefix when it isn't empty. #2235
+* [ENHANCEMENT] [Webhook] Add `max_alerts` option to limit the number of alerts included in the payload. #2274
```

Comment:
When do we use `[Webhook]` vs. just mentioning the component type in the sentence? E.g. for the Pagerduty change above, it's just part of the sentence.

Reply:
Thanks for spotting! I try to mention `[component]` so that folks using it notice it more easily.